docs/assets/versions.js

```diff
@@ -80,11 +80,20 @@ function sortVersionLinks(container) {
   const dlElements = container.querySelectorAll('dl');
 
   dlElements.forEach(dl => {
-    const dt = dl.querySelector('dt');
-    if (dt && dt.textContent.trim().toLowerCase() === 'versions') {
-      // Found the Versions <dl>
-      const ddElements = Array.from(dl.querySelectorAll('dd'));
+    const ddElements = Array.from(dl.querySelectorAll('dd'));
+
+    // Check if ddElements contain version links
+    const isVersionDl = ddElements.some(dd => {
+      const link = dd.querySelector('a');
+      if (link) {
+        // Check if the link's href matches version pattern
+        return /\/en\/(release-(?:v?\d+[\.\d+]*)|\d+\.\d+\.\d+|latest|stable)\//.test(link.getAttribute('href'));
```

Comment:
https://github.com/argoproj/argo-cd/blob/3cb9292dff006289fc3c74f8a5858ab1af51c6d6/docs/assets/versions.js#L23

![image](https://github.com/user-attachments/assets/e795afbe-1fbe-4841-9244-5ebc6c8d7637)

The version regex expression Line 23 excludes versions without the release- prefix, except for 'latest' or 'stable'. However,Here currently including them. Why is this necessary? 🤔

![image](https://github.com/user-attachments/assets/35cabb2d-155c-42b7-8ac8-e84a3b164b2d)

Reply:
I have now corrected it. I appreciate your help!